### ISSUE TITLE
No such var: ana/forms-seq* on 0.5.0

### ISSUE THREAD

**Issue by snoe** [bhauman/lein-figwheel#286]
similar to #131 I'm getting the error with clojurescript 1.7.170 and figwheel 0.5.0

figwheel 0.4.1 is fine

Stacktrace and project below

```
(defproject nvim-parinfer "0.1.0-SNAPSHOT"
  :description "A neovim parinfer plugin"
  :url "http://github.com/snoe/nvim-parinfer.js"

  :dependencies [[org.clojure/clojure "1.7.0"]
                 [org.clojure/clojurescript "1.7.170"]
                 [parinfer "0.1.0-SNAPSHOT"]
                 [org.clojure/core.async "0.2.374"]]

  :plugins [[lein-cljsbuild "1.1.1"]
                [lein-figwheel "0.5.0"]]

  :source-paths ["src" "target/classes"]

  :test-paths ["test"]

  :clean-targets ["lib/nvim-parinfer" "lib/nvim-parinfer.js"]

  :figwheel {:server-port 9443}

  :cljsbuild {:builds [{:id "plugin"
                        :source-paths ["src"]
                        :figwheel true
                        :compiler {:main nvim-parinfer.main
                                   :asset-path "lib/nvim-parinfer"
                                   :hashbang false
                                   :output-to "lib/nvim-parinfer.js"
                                   :output-dir "lib/nvim-parinfer"
                                   :optimizations :simple
                                   :target :nodejs
                                   :cache-analysis true
                                   :source-map true}}
                       {:id "fig-test"
                        :source-paths ["src" "test"]
                        :figwheel true
                        :compiler {:main nvim-parinfer.fig-test
                                   :output-to "target/out/tests.js"
                                   :output-dir "target/out"
                                   :target :nodejs
                                   :optimizations :none
                                   :source-map true}}]})
```

```
$ lein figwheel fig-test
clojure.lang.Compiler$CompilerException: java.lang.RuntimeException: No such var: ana/forms-seq*, compiling:(figwheel_sidecar/utils.clj:49:21)
 at clojure.lang.Compiler.analyze (Compiler.java:6464)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$InvokeExpr.parse (Compiler.java:3665)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6646)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.access$100 (Compiler.java:38)
    clojure.lang.Compiler$LetExpr$Parser.parse (Compiler.java:6050)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler$TryExpr$Parser.parse (Compiler.java:2191)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler$LetExpr$Parser.parse (Compiler.java:6100)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$IfExpr$Parser.parse (Compiler.java:2708)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler$TryExpr$Parser.parse (Compiler.java:2191)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler$FnMethod.parse (Compiler.java:5217)
    clojure.lang.Compiler$FnExpr.parse (Compiler.java:3846)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6642)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.access$100 (Compiler.java:38)
    clojure.lang.Compiler$DefExpr$Parser.parse (Compiler.java:538)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler.eval (Compiler.java:6707)
    clojure.lang.Compiler.load (Compiler.java:7130)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.loadResourceScript (RT.java:361)
    clojure.lang.RT.load (RT.java:440)
    clojure.lang.RT.load (RT.java:411)
    clojure.core$load$fn__5066.invoke (core.clj:5641)
    clojure.core$load.doInvoke (core.clj:5640)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invoke (core.clj:5446)
    clojure.core$load_lib$fn__5015.invoke (core.clj:5486)
    clojure.core$load_lib.doInvoke (core.clj:5485)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$load_libs.doInvoke (core.clj:5524)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$require.doInvoke (core.clj:5607)
    clojure.lang.RestFn.invoke (RestFn.java:551)
    figwheel_sidecar.config$eval2752$loading__4958__auto____2753.invoke (config.clj:1)
    figwheel_sidecar.config$eval2752.invoke (config.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:6703)
    clojure.lang.Compiler.eval (Compiler.java:6692)
    clojure.lang.Compiler.load (Compiler.java:7130)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.loadResourceScript (RT.java:361)
    clojure.lang.RT.load (RT.java:440)
    clojure.lang.RT.load (RT.java:411)
    clojure.core$load$fn__5066.invoke (core.clj:5641)
    clojure.core$load.doInvoke (core.clj:5640)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invoke (core.clj:5446)
    clojure.core$load_lib$fn__5015.invoke (core.clj:5486)
    clojure.core$load_lib.doInvoke (core.clj:5485)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$load_libs.doInvoke (core.clj:5524)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$require.doInvoke (core.clj:5607)
    clojure.lang.RestFn.invoke (RestFn.java:457)
    leiningen.figwheel$eval2703$loading__4958__auto____2704.invoke (figwheel.clj:1)
    leiningen.figwheel$eval2703.invoke (figwheel.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:6703)
    clojure.lang.Compiler.eval (Compiler.java:6692)
    clojure.lang.Compiler.load (Compiler.java:7130)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.loadResourceScript (RT.java:361)
    clojure.lang.RT.load (RT.java:440)
    clojure.lang.RT.load (RT.java:411)
    clojure.core$load$fn__5066.invoke (core.clj:5641)
    clojure.core$load.doInvoke (core.clj:5640)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invoke (core.clj:5446)
    clojure.core$load_lib$fn__5015.invoke (core.clj:5486)
    clojure.core$load_lib.doInvoke (core.clj:5485)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$load_libs.doInvoke (core.clj:5524)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$require.doInvoke (core.clj:5607)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    leiningen.core.utils$require_resolve.invoke (utils.clj:68)
    leiningen.core.utils$require_resolve.invoke (utils.clj:71)
    leiningen.core.main$lookup_task_var.invoke (main.clj:61)
    leiningen.core.main$pass_through_help_QMARK_.invoke (main.clj:68)
    leiningen.core.main$task_args.invoke (main.clj:71)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:308)
    leiningen.core.main$_main$fn__6136.invoke (main.clj:377)
    leiningen.core.main$_main.doInvoke (main.clj:366)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:624)
    clojure.main$main_opt.invoke (main.clj:315)
    clojure.main$main.doInvoke (main.clj:420)
    clojure.lang.RestFn.invoke (RestFn.java:482)
    clojure.lang.Var.invoke (Var.java:401)
    clojure.lang.AFn.applyToHelper (AFn.java:171)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
```

**Comment by bhauman** [comment]
You are going to need to do a lein clean.

For some reason you are getting an old version of ClojureScript compiling your project.

**Comment by snoe** [comment]
lein clean didn't do it but it turned out I needed to upgrade leiningen from 2.5.0 to 2.5.3

**Comment by escherize** [comment]
Thanks, I had this same issue.

**Comment by danielcompton** [comment]
We had the same issue when upgrading. What fixed it for us was removing `[com.cemerick/clojurescript.test "0.3.3"]` from our `:plugins`.

**Comment by danielcompton** [comment]
@bhauman is it possible that Figwheel is compiling in the Leiningen process, not the application process? We only got these errors when running figwheel, cljsbuild was able to run fine.

**Comment by dbleonard** [comment]
Some additional information from my experiences with the problem when I upgraded from 0.4.1 to 0.5.0:
My env
`:dependencies [
  [org.clojure/clojure "1.7.0"]
  [org.clojure/clojurescript "1.7.228" :scope "provided"]
  [lein-figwheel "0.5.0-3"]
  [com.cemerick/clojurescript.test "0.3.3"]
...]`

 I tried a `lein clean` to no avail.  I then removed `[com.cemerick/clojurescript.test "0.3.3"]` per @danielcompton, ran `lein clean`, and `lein figwheel` began to work.  I then added test back in to attempt to replicate the failure, ran a `lein clean`, and when I ran `lein figwheel` expecting failure, it actually worked.  Incidentally, `lein cljsbuild` and `lein repl` never failed for me, either.

**Comment by dbleonard** [comment]
I should add that I was running Leiningen 2.5.3 already when I experienced problems.

**Comment by DanielJomphe** [comment]
> Another way to fix it is to add an explicit dependency on 1.7.170 in your :plugins.

Since Google led me twice or thrice here in the last few months in two different projects which encountered this issue, I just wanted to chime in and inform people they should try heeding this recommendation as it's the only one that worked for me in both cases.

Thus, in `project.clj`:
```clj
  :plugins [[lein-cljsbuild "1.1.1"]
            [lein-figwheel "0.5.0-2"]
            [org.clojure/clojurescript "1.7.170"]  ; <== This is what solves it for me
  ]
```

And obviously, you might be using a more recent version of CLJS in your project than that one, so conform it to what you use.
